packages/wrangler/src/kv.ts

```diff
@@ -205,15 +206,38 @@ export async function deleteKVKeyValue(
   );
 }
 
+/**
+ * Formatter for converting e.g. 5,328 --> 5.3K
+ */
+const compact = new Intl.NumberFormat("en-US", {
+  compactDisplay: "short",
+  notation: "compact",
+}).format;
+
+/**
+ * Helper function for bulk requests, logs ongoing output to console.
+ */
+function logBulkProgress(
+  request: "put" | "delete",
```

Comment:
NIT: 
```suggestion
  operation: "put" | "delete",
```

or something similar?